robocop/utils/pylama_parse.py

```diff
@@ -0,0 +1,8 @@
+"""Run Pylama and parse output to find errors and exit process with proper return code"""
+import re
+import sys
+import subprocess
+
+output = subprocess.run('python -m pylama -o pylama.ini', stdout=subprocess.PIPE)
```

Comment:
what if errors are reported to std.error? Fatal errors like missing files, wrong config are reposted to stderr instead

Reply:
I changed the stderr stream to be directed to stdout now.